src/pages/apps/[id]/environments/[env-id]/snippets/_components/SnippetModal.js

```diff
@@ -10,6 +10,8 @@ import InfoBox from "~/components/InfoBox";
 import Button from "~/components/Button";
 import { connect } from "~/utils/context";
 import { upsertSnippets } from "../actions";
+import "codemirror/keymap/sublime";
+import "codemirror/theme/idea.css";
```

Comment:
**thought** @eldemcan since the code editor is only used in the Snippet Modal I think it's fine for now to leave this css import here. But later when we create an abstraction for the Code component we should remember to move this import there as well.